examples/graphql-server-typeorm-postgres/README.md

```diff
@@ -0,0 +1,61 @@
+# typeorm-server-typescript
+
+This example demonstrate how to use [accounts-js](https://github.com/accounts-js/accounts) with PostgreSQL thanks to the amazing @accountsJS/typeorm plugin built by @birkir for his great PrimeCMS.
```

Comment:
it's `@accounts/typeorm` instead of `@accountsJS/typeorm` :)

Reply:
Yeah sorry, I totally misspelled that one when edit rushing the MD file, will fix it.